### PR TITLE
refactor(empty): less to cssinjs

### PR DESCRIPTION
**Pull request by buqiyuan**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [x] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: components/empty/simple.tsx [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
-      <g transform="translate(0 1)" fill="none" fill-rule="evenodd">
-        <ellipse class={`${prefixCls}-ellipse`} fill="#F5F5F5" cx="32" cy="33" rx="32" ry="7" />
-        <g class={`${prefixCls}-g`} fill-rule="nonzero" stroke="#D9D9D9">
-          <path d="M55 12.76L44.854 1.258C44.367.474 43.656 0 42.907 0H21.093c-.749 0-1.46.474-1.947 1.257L9 12.761V22h46v-9.24z" />
-          <path
-            d="M41.613 15.931c0-1.605.994-2.93 2.227-2.931H55v18.137C55 33.26 53.68 35 52.05 35h-40.1C10.32 35 9 33.259 9 31.137V13h11.16c1.233 0 2.227 1.323 2.227 2.928v.022c0 1.605 1.005 2.901 2.237 2.901h14.752c1.232 0 2.237-1.308 2.237-2.913v-.007z"
-            fill="#FAFAFA"
-            class={`${prefixCls}-path`}
-          />
+    const { colorFill, colorFillTertiary, colorFillQuaternary, colorBgContainer } = token.value;
```
Comment (tangjinzhou) [inline components/empty/simple.tsx:9]: 直接结构出来就失去响应式了

File: components/empty/index.tsx [hunk 2]
```diff
@@ -25,61 +28,59 @@ export interface EmptyProps {
...
+    const [wrapSSR, hashId] = useStyle(prefixClsRef);
 
-  return (
-    <LocaleReceiver
-      componentName="Empty"
-      children={(locale: Locale) => {
-        const des = typeof description !== 'undefined' ? description : locale.description;
-        const alt = typeof des === 'string' ? des : 'empty';
-        let imageNode: EmptyProps['image'] = null;
+    const {
```
Comment (tangjinzhou) [inline components/empty/index.tsx:39]: 直接结构出来就失去响应式了
原来是函数式组件，不存在这个问题，你现在改成普通组件，就会问题了，放在return 里面

File: components/empty/index.tsx [hunk 3]
```diff
@@ -25,61 +28,59 @@ export interface EmptyProps {
...
-interface EmptyType extends FunctionalComponent<EmptyProps> {
-  displayName: string;
-  PRESENTED_IMAGE_DEFAULT: VueNode;
-  PRESENTED_IMAGE_SIMPLE: VueNode;
-}
-
-const Empty: EmptyType = (props, { slots = {}, attrs }) => {
-  const { direction, prefixCls: prefixClsRef } = useConfigInject('empty', props);
-  const prefixCls = prefixClsRef.value;
+const Empty = defineComponent({
```
Comment (tangjinzhou) [inline components/empty/index.tsx:31]: 定义 props
添加 inheritAttrs: false,

File: components/empty/index.tsx [hunk 4]
```diff
@@ -25,61 +28,59 @@ export interface EmptyProps {
...
+  name: 'AEmpty',
+  setup(props, { slots = {}, attrs }) {
+    const { direction, prefixCls: prefixClsRef } = useConfigInject('empty', props);
+    const prefixCls = prefixClsRef.value;
```
Comment (tangjinzhou) [inline components/empty/index.tsx:35]: 失去响应式了

Comment (tangjinzhou) [reply components/empty/index.tsx:35]: 原来是函数式组件，不存在这个问题，你现在改成普通组件，就会问题了，放在return 里面

**Review by tangjinzhou — CHANGES_REQUESTED** [review]

File: components/empty/simple.tsx (hunk 1 above)
Comment (buqiyuan) [reply components/empty/simple.tsx]: 我的，粗心了

**Review by tangjinzhou — APPROVED** [review]

**Pull request merged**